sources/packages/forms/Dockerfile.dev

```diff
@@ -31,4 +31,4 @@ ENV DEBUG=""
 
 # This will initialize the application based on
 # some questions to the user (login email, password, etc.)
-ENTRYPOINT [ "node", "main" ]
+ENTRYPOINT [ "node", "--no-node-snapshot", "main" ]
```

Comment:
Can you please provide more details on "changed to enable development environment login"?
Is it related to the Redis changes?

Reply:
no not redis related, i can remove it.